elisp/ghc-process.el

```diff
@@ -27,6 +27,7 @@
 
 (defvar ghc-command "ghc-mod")
 
+(defvar report-ghc-errors t "Report GHC errors to *GHC Error* buffer")
```

Comment:
Elisp convention is to prefix everything with a namespace, in our case that's `ghc` so this should be `ghc-report-error`.